packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -578,6 +583,9 @@ export interface IFluidDataStoreContext extends IFluidParentContext {
 	/**
 	 * @deprecated 0.16 Issue #1635, #3631
 	 */
+	// Seems like this can be removed now; the issues mentioned in the @deprecated tag are about _createDataStoreWithProps
```

Comment:
Any idea who could answer this question? Should we file a work item to follow up on this (and reference in the comment)?

Reply:
I was planning to ask @tyler-cai-microsoft since he removed `_createDataStoreWithProps`, maybe @markfields .